### PR TITLE
feat: add support for darwin/arm64 arch

### PR DESCRIPTION
**Pull request by lander2k2**
Signed-off-by: Rich Lander <lander2k2@protonmail.com>

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -78,19 +80,19 @@ jobs:
           - name: Standalone Operator
             artifact: standalone-codebase
             test-workload-path: test/cases/standalone
-            go-version: 1.17
+            go-version: 1.19
```
Comment (scottd018) [inline .github/workflows/test.yaml:83]: @lander2k2 we should not change all of the `go-version` lines to 1.19.  This would effectively test all cases (standalone, collection, edge-standalone, edge-collection) twice with the same version of go.  The purpose of having 2 different tests for different versions is that we should be supporting 2 versions of go (current, and 1 prior release) because not everyone will upgrade to the latest version right away.

Suggestion is that the edge cases should be tested with the previous version (1.18).  This would mean that lines 87, 95, 155, and 165 should read `go-version: 1.18`.  This would allow us coverage for testing both functional and e2e tests would be supported for both versions of go.

**Review by scottd018 — CHANGES_REQUESTED** [review]
